src/components/app/details/triggerView/workflow.service.ts

```diff
@@ -127,11 +146,16 @@ export function processWorkflow(
             s.width = dimensions.staticNodeSizes.nodeWidth
         })
 
-        ciNode.x =
-            startX +
-            workflowOffset.offsetX +
-            dimensions.staticNodeSizes.nodeWidth +
-            dimensions.staticNodeSizes.distanceX
+        if (ciNode.type === 'WEBHOOK') {
```

Comment:
Can we create const/enum type for CI/CD/WEBHOOK?

Reply:
Done